lib/plugins/create/templates/aws-nodejs-typescript/webpack.config.js

```diff
@@ -8,6 +8,7 @@ Object.keys(slsw.lib.entries).forEach(key => (
 ));
 
 module.exports = {
+  mode: 'production',
```

Comment:
This should be `slsw.lib.webpack.isLocal ? 'development' : 'production'` to enable debugging features when running locally.